lib/engine/game/g_18_uruguay/step/buy_sell_par_shares.rb

```diff
@@ -9,9 +9,17 @@ module Step
         class BuySellParShares < Engine::Step::BuySellParShares
           def can_buy?(entity, bundle)
             return false if bundle&.corporation == @game.rptla && !@game.phase.status.include?('rptla_available')
+            return true if loan_limit(entity, bundle)
```

Comment:
The usual way to allow a player to go above the normal corporation ownership limit is by overriding `can_gain?`, rather than changing the behaviour of `can_buy?`.

Reply:
Changed to can_gain?